movie_webscraper.py

```diff
@@ -0,0 +1,49 @@
+import bs4
+import requests
+import json
+import random
+
+# Download IMDB's Top 250 data
+
+url = 'http://www.imdb.com/chart/top'
+response = requests.get(url)
+soup = bs4.BeautifulSoup(response.text, 'lxml')
+
+movies = soup.select('td.titleColumn')
+links = [a.attrs.get('href') for a in soup.select('td.titleColumn a')]
+crew = [a.attrs.get('title') for a in soup.select('td.titleColumn a')]
+ratings = [b.attrs.get('data-value')
+          for b in soup.select('td.posterColumn span[name=ir]')]
+
+imdb = []
+
+# Store each item into dictionary (data), then put those into a list (imdb)
+for index in range(0, len(movies)):
+    movie_string = movies[index].get_text()
+    movie = (' '.join(movie_string.split()).replace('.', ''))
+    movie_title = movie[len(str(index))+1:-7]
+    place = movie[:len(str(index))-(len(movie))]
+    # data = {"movie_title": movie_title, "place": place}
+    imdb.append(movie_title)
+
+# THIS WILL PRINT THE ENTIRE 250 LIST
+# for item in imdb:
+# print(item['place'], '-', item['movie_title'])
+# print(item['movie_title'])
+
+random_movies = random.sample(imdb, 10)
```

Comment:
This will limit our return of movies to 10. Keeping it consistent with the number of books that are returned (also 10)

Reply:
I refactored the events API call down to five, but I think this is okay to make it easier to work with.